### PR TITLE
[jsk_data] Add common rosbag_record and play file for fetch

### PR DESCRIPTION
**Pull request by iory**
This PR addes common record and play file for fetch robot.
You can record bagfile.
```
roslaunch jsk_data fetch_record.launch
```

In default, rosbag saved at /tmp/fetch_rosbag/ .

### CODE REVIEW

**Comment by k-okada** [comment]
@708yamaguchi please fix by youself and send PR

File: jsk_data/launch/fetch_play.launch [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+          pkg="rosbag" type="play"
+          launch-prefix="xterm -e"
+          args="$(arg rosbag_option) $(arg bagfile_names)" />
+  </group>
+  <group unless="$(arg use_xterm)">
+    <node name="rosbag_play"
+          pkg="rosbag" type="play"
+          args="$(arg rosbag_option) $(arg bagfile_names)" />
+  </group>
+
```
[the remark this answers is not in the document]
Comment (708yamaguchi) [reply jsk_data/launch/fetch_play.launch]: I added.

File: jsk_data/launch/fetch_play.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+filenames="";
+for filename in ${@:1:$#}
+do
+    filenames=$filenames" "$filename;
+done
+echo $filenames
+
+roslaunch jsk_data fetch_play.launch bagfile_names:="$filenames";
```
[the remark this answers is not in the document]
Comment (708yamaguchi) [reply jsk_data/launch/fetch_play.sh]: I added.

**Comment by 708yamaguchi** [comment]
I added commits.

I checked this PR by recording and playing rosbag of fetch.

**Pull request merged**